js/blocks/SensorsBlocks.js

```diff
@@ -362,7 +362,7 @@ function setupSensorsBlocks() {
             ) {
                 logo.statusFields.push([blk, "toascii"]);
             } else {
-                let cblk1 = logo.blocks.blockList[blk].connections[1];
+                var cblk1 = logo.blocks.blockList[blk].connections[1];
```

Comment:
Why did you switch this back to var?

Reply:
@walterbender  because if I put there let , I am getting branch conflicts.

Reply:
What do you mean by "branch conflicts"?

Reply:
@walterbender I meant that this branch would have conflicts with master branch (Conflicts that usually happen in a PR). I don't know why that is happening if I put let in place of var.

Reply:
I will sort out the merge conflict. Please make the code itself correct.  And please be sure to test..

Reply:
@walterbender  It is Done!!

Reply:
Also I have tested it already.